components/ui/useCode/StepTwo.tsx

```diff
@@ -0,0 +1,84 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as z from "zod";
+import { useForm } from "react-hook-form";
+import { Button } from "@/components/ui/button";
+import { Form } from "@/components/ui/form";
+import useStepperStore from "@/data-store/stepper-store";
+import { FormSubmitting } from "./Submitting";
+import useSessionStore from "@/data-store/session-store";
+import LandingPageCode from "@/components/landing/test-challenges/placeholder-code";
+import { useRouter } from "next/navigation";
+import { ChallengeFormField } from "./Challenge-FormField";
+
+const formStepTwoSchema = z.object({
+    challenge: z.enum(
+        ["helloWorld", "simpleDialog", "simpleNavBar", "brightSunnyDay"],
+        {
+            required_error: "Please select a challenge",
+        }
+    ),
+});
+
+export function StepTwo() {
+    const PROGRESS_INCREMENT: number = 33;
```

Comment:
Put it into the zustand store. You've repeated the exact same line already.

Reply:
done. I put it and a few others as consts outside of the actual zustand store, but in the same file. If you think this is bad I can explain why I did it in a call. Because some of the consts I created depended on other defined variables, it didnt work inside the zustand store structure, so they were made to be const variables outside of it.